x-pack/plugins/synthetics/public/apps/synthetics/components/common/components/location_status_badges.tsx

```diff
@@ -109,7 +109,7 @@ const MonitorDetailLinkForLocation = ({
 
   return (
     <EuiBadge
-      iconType={() => <EuiIcon size="s" type="dot" color={color} />}
+      iconType={() => <EuiIcon size="m" type="dot" color={color} />}
```

Comment:
In the designs, the dot size is `6x6` i.e. `size-"s"`. `size="m"` makes it `8x8`.